lib/project_types/script/layers/infrastructure/assemblyscript_dependency_manager.rb

```diff
@@ -41,15 +41,13 @@ def write_package_json
               "version": "1.0.0",
               "devDependencies": {
                 "@shopify/scripts-sdk-as": "#{@extension_point.sdks[:ts].sdk_version}",
+                "@shopify/scripts-toolchain-as": "#{@extension_point.sdks[:ts].toolchain_version}",
                 "#{@extension_point.sdks[:ts].package}": "#{@extension_point.sdks[:ts].version}",
                 "@as-pect/assembly": "3.1.1",
                 "@as-pect/cli": "3.1.1",
                 "@as-pect/core": "3.1.1",
-                "yargs": "10.0.0",
                 "as-wasi": "^0.0.1",
                 "assemblyscript": "^0.9.4",
```

Comment:
We'll need to remove the comma here or else the json parser will blow up when trying to install deps

```suggestion
                "assemblyscript": "^0.9.4"
```